tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/document/documents.controller.js

```diff
@@ -286,6 +286,35 @@
 
         };
 
+        vm.showDocumentAsPinOrUnpin = function (searchRequest, response) {

```

Comment:
Added one reusable method which will be called at several places for pin and unpin functionality